archetypes/helidon/src/main/archetype/common/packaging.xml

```diff
@@ -92,6 +92,36 @@ kubectl delete -f app.yaml
                     </templates>
                 </output>
             </boolean>
+            <boolean id="jpms-se"
```

Comment:
Move these inputs under each flavor so that the ids are flavor less.

Reply:
Done